src/backend/InvenTree/plugin/urls.py

```diff
@@ -16,8 +19,35 @@ def get_plugin_urls():
     urls = []
 
     if get_global_setting('ENABLE_PLUGINS_URL', False) or settings.PLUGIN_TESTING_SETUP:
-        for plugin in registry.plugins.values():
-            if plugin.mixin_enabled(PluginMixinEnum.URLS):
-                urls.append(plugin.urlpatterns)
+        for plugin in registry.with_mixin(PluginMixinEnum.URLS):
+            try:
+                if plugin_urls := plugin.urlpatterns:
+                    # Check if the plugin has a custom URL pattern
+                    for url in plugin_urls:
+                        # Attempt to resolve against the URL pattern as a validation check
+                        try:
+                            url.resolve('')
+                        except Resolver404:
+                            pass
```

Comment:
What are you trying to protect again bat here? This statement has seldom an effect

Reply:
It will at least ensure that each returned URL is a valid URLpattern, is the thinking